extensions/amp-story-dev-tools/0.1/amp-story-dev-tools-tab-preview.css

```diff
@@ -0,0 +1,243 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.i-amphtml-story-dev-tools-device {
+  background: #101010 !important;
+  border-radius: 40px !important;
+  box-shadow: 0px 11.0046px 27.5114px rgba(0, 0, 0, 0.34), inset 0px 2.20091px 3.30137px rgba(44, 46, 52, 0.6) !important;
+  display: flex !important;
+  flex-direction: column !important;
+  overflow: hidden !important;
+  position: absolute !important;
+  margin: auto !important;
+  height: fit-content !important;
+  top: 0 !important;
+  bottom: 0 !important;
+  padding: 10px 8px !important;
+  height: fit-content !important;
+  opacity: 0;
+  transition: opacity 0.15s ease-in-out;
+}
+
+.i-amphtml-story-dev-tools-device-screen {
+  border-radius: 30px !important;
+  background: #2E3137 !important;
+  overflow: hidden !important;
+  padding-top: 40px;
+}
+
+amp-story-player a {
+  display: none !important;
+}
+
+.i-amphtml-story-dev-tools-device-chips-container {
+  position: absolute !important;
+  bottom: 0 !important;
+  width: 100% !important;
+  text-align: center !important;
+}
+
+.i-amphtml-story-dev-tools-device-chip {
+  border-radius: 16px !important;
+  background: #2e2e2e !important;
+  border: 1px solid #2e2e2e !important;
+  height: 32px !important;
+  font-size: 13px !important;
+  line-height: 20px !important;
+  display: inline-flex !important;
+  align-items: center !important;
+  padding: 0 6px 0 10px !important;
+  margin: 0 5px !important;
+  max-width: 300px !important;
+  transition: background 0.2s ease-in-out, color 0.2s ease-in-out, max-width 0.2s ease-in-out !important;
+  white-space: nowrap !important;
+  cursor: pointer !important;
+}
+
+.i-amphtml-story-dev-tools-device-chip:hover {
+  background: #3d3d3d !important;
+}
+
+.i-amphtml-story-dev-tools-device-chip svg {
+  cursor: pointer !important;
+  background: black !important;
+  border-radius: 50% !important;
+  padding: 2px !important;
+  margin-left: 10px !important;
+  transform: rotate(0deg) !important;
+  transition: transform 0.2s ease-in-out !important;
+}
+
+.i-amphtml-story-dev-tools-device-chips .i-amphtml-story-dev-tools-device-chip[inactive] {
```

Comment:
`disabled` is a more commonly used attribute

Reply:
Disabled would mean you can't click on the chip right? Here we want to mark it as the device not being shown.

We'll want to mark the chips as `disabled` in a separate PR when you already have 4 devices on the screen to prevent users from adding more devices, so I think we can save that attribute for that scenario. WDYT?